client/src/components/MultiJurisdictionAudit/Setup/Contests/index.tsx

```diff
@@ -252,21 +240,28 @@ const Contests: React.FC<IProps> = ({
                             contestIndex={i}
                           />
                         </FormSection>
+                        {values.contests.length > 1 && (
+                          <FormButton
+                            intent="danger"
+                            onClick={() => contestsArrayHelpers.remove(i)}
+                          >
+                            Remove Contest {i + 1}
+                          </FormButton>
+                        )}
```

Comment:
wonder if wrapping each contest in a card would make it easier to parse for the user. right now it's a lot of fields that aren't super separated visually

Reply:
@jonahkagan There's an `<hr/>`, but yeah, it could be more visually distinct. There's likely a satisfactory wrapping component in the blueprintjs uikit that would work.